### DISCUSSION TITLE
How long did i stay at 1 location

### DISCUSSION THREAD

**Discussion by pohmervin** [teslamate-org/teslamate#4167] [Ideas]
I have a feature request.

I would like to know how long i've spent at my office. Its helpful to me because i pay for season parking. And my office hours are not fixed. If im below a certain hours, it will actually be cheaper for me to not pay for season parking.

If i will to calculate it manually i have to pull all my drives timing that ends in office and then calculate how long before i leave the office. Is there a way for me to do it easier?

**Comment by eliluong** [comment]
Without having to create a new custom dashboard, if you go to the Drives dashboard, you can click to get this menu:

![image](https://github.com/user-attachments/assets/5c761e07-68c8-42cd-b0cf-ceda64a6be25)

Then go to Data. Then you can export as CSV. Then you just need to look at start_path and end_path, or start_address and end_address. With this data you should be able to calculate what you need.

**Reply by pohmervin** [reply]
Thanks, after i filtered office, it makes it easier to calculate. still would love a function for that though. but for now its helpful! thanks again
